crates/aptos/src/move_tool/mod.rs

```diff
@@ -163,10 +164,10 @@ impl FrameworkPackageArgs {
         addresses: BTreeMap<String, ManifestNamedAddress>,
         prompt_options: PromptOptions,
     ) -> CliTypedResult<()> {
-        const APTOS_FRAMEWORK: &str = "AptosFramework";
-        const APTOS_GIT_PATH: &str = "https://github.com/aptos-labs/aptos-core.git";
-        const SUBDIR_PATH: &str = "aptos-move/framework/aptos-framework";
-        const DEFAULT_BRANCH: &str = "mainnet";
+        const SUPRA_FRAMEWORK: &str = "SupraFramework";
+        const SUPRA_FRAMEWORK_GIT_PATH: &str = "https://github.com/Entropy-Foundation/aptos-core.git";
+        const SUBDIR_PATH: &str = "aptos-move/framework/supra-framework";
+        const DEFAULT_BRANCH: &str = "integrate_consensus_key";
```

Comment:
Can we refactor somehow so that name of the branch/path/repo etc, instead of being hardcoded in the repo, is in some config file?

Reply:
As per the discussion this change can be skipped as we do not want user to play with this config.